config/webhook/manifests.yaml

```diff
@@ -52,6 +22,7 @@ webhooks:
     operations:
     - CREATE
     - UPDATE
+    - DELETE
     resources:
-    - applications
```

Comment:
Dunno why it was application before, I also don't know why validation webhook has been configured without any validation code.